### DISCUSSION TITLE
Power bargraph moves too fast

### DISCUSSION THREAD

**Discussion by IU4JNR** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#217] [Ideas]
I use alternately a sunsdr2 pro and a sunsdr2dx. Both are connected to a Acom 600s PA which i drive with about 10-15w. In both cases the bargraph becomes useless:
In Sunsdr2 pro The needle movement is frantic and unreadable; i suggest to make the movement a little slower to make the reading easier.
In Sunsdr2 Dx The needle movement is too little and unreadable. I suggest  to make it scalable  based on the level of power setup.
Max Iu4jnr

**Comment by N8SDR007** [comment]
Since the newer versions of V3 now has both RMS and peak (from what I can tell on the bar graph) it would be very nice adding to Max suggestion above to have a peak hold timer option! where perhaps a PEAK is help for X value adjustable by end user
